.gitignore

```diff
@@ -17,3 +17,6 @@ cfnnag_output
 /sample_file/
 /src/layers/umccr_utils/python/
 .aws-sam
+
+.yarn
```

Comment:
Good move to localised yarn v4, Will. This git ignore looks a bit wide.

The localised yarn (v4 or latest) script has to check it into the repo. Hence, localised.

There should be couple of folder need exclusion from git ignore: e.g. `!.yarn/releases` (where localised yarn script installed) and `!.yarn/plugins` (any extensions use). So on so forth.

See Portal example:
- https://github.com/umccr/data-portal-apis/tree/dev/.yarn
- https://github.com/umccr/data-portal-apis/blob/dev/.gitignore#L45-L52

And here is pointer to doc:
- https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Reply:
Ahh thanks for this! Will update it